pkg/platform/kube/logProxy/elastic.go

```diff
@@ -60,10 +66,222 @@ func NewElasticLogProxy(config *platformconfig.ElasticSearchConfig) (*ElasticLog
 	return esClient, err
 }
 
+func (e *ElasticLogProxy) GetFunctionReplicas(ctx context.Context, options *GetFunctionReplicaOptions) ([]string, error) {
+	if e.client == nil {
+		return nil, errors.New("Elasticsearch client is not configured")
+	}
```

Comment:
related to my previous comments - The caller won't necessarily want to fail if the client is not configured.